startup.sh

```diff
@@ -226,5 +243,5 @@ echo -e "Use 'docker service ls' to check status of services."
 echo -e "Each service has REPLICAS 1/1 when everything works (it may take several minutes to start all services)."
 echo
 echo -e "Use 'docker stack rm $stackName' to stop all services and"
-echo -e "'docker volume prune' to remove old data if needed."
+echo -e "'${scriptName} --delete-volumes' to remove old data if needed."
```

Comment:
we could create "teardown.sh -v" where we can put:
1. docker stack rm $stackName
2. wait until all containers are down
3. delete volumes (if -v was set)

Reply:
teardown.sh is prepared and can be tested